migrations/versions/0317_add_provider_response.py

```diff
@@ -0,0 +1,21 @@
+"""
+
+Revision ID: 0317_add_provider_response
+Revises: 0316_now_live_template
+Create Date: 2021-04-14 10:00:42.383782
+
+"""
+
+revision = '0317_add_provider_response'
+down_revision = '0316_now_live_template'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('notifications', sa.Column('provider_response', sa.Text(), nullable=True))
```

Comment:
does this mean that this table will be filled up with the same text over and over again from the provider? I guess it's overly complicated to suggest an enum/choice and serialize it as an int into the database?
I guess this status, e.g. NOTIFICATION_TECHNICAL_FAILURE is used everywhere else as well, so that's the problem...

Reply:
This column will be filled for SMS with a technical failure status. We get less than 200 errors/month at the moment and this table only holds 7 days of data. As you pointed out, other status fields like the status store strings already which I guess is fine considering how cheap database storage is nowadays.